src/WtsApi32.Desktop/WtsApi32+WtsConnectStateClass.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Runtime.InteropServices;
+    using System.Text;
+
+    /// <content>
+    /// Exported functions from the WtsApi32.dll Windows library
+    /// that are available to Desktop apps only.
+    /// </content>
+    public static partial class WtsApi32
+    {
+        /// <summary>
+        ///     Specifies the connection state of a Remote Desktop Services session.
+        /// </summary>
+        public enum WtsConnectStateClass
```

Comment:
We currently use the language norm (PascalCase) when we create enums but keep the original names if they exists.

As the 2 enums you cited aren't named in the origial source code (They are just constants) we choose a name :
* https://msdn.microsoft.com/en-us/library/windows/desktop/ms684863(v=vs.85).aspx
* https://msdn.microsoft.com/en-us/library/windows/desktop/ms682013(v=vs.85).aspx

The contributing guide allude to it but maybe we can make it clearer : https://github.com/AArnott/pinvoke/blob/master/CONTRIBUTING.md#naming

> There is a tension between keeping names consistent between native and managed code, and conforming to common .NET naming patterns such as camelCase and PascalCase. For this project, **we preserve names of methods, parameter values, constants, and anything else found in native header files for these reasons**: